larpandoracontent/LArShowerRefinement/ProtoShowerMatchingTool.cc

```diff
@@ -213,22 +211,43 @@ bool ProtoShowerMatchingTool::AreDirectionsConsistent(CartesianVector directionU
     if (directionV.GetX() * directionW.GetX() < 0.f)
         return false;
 
-    const CartesianVector projectionU(LArGeometryHelper::MergeTwoDirections(this->GetPandora(), TPC_VIEW_V, TPC_VIEW_W, directionV, directionW));
-    const CartesianVector projectionV(LArGeometryHelper::MergeTwoDirections(this->GetPandora(), TPC_VIEW_W, TPC_VIEW_U, directionW, directionU));
-    const CartesianVector projectionW(LArGeometryHelper::MergeTwoDirections(this->GetPandora(), TPC_VIEW_U, TPC_VIEW_V, directionU, directionV));
+    const CartesianVector xAxis(1.f, 0.f, 0.f);
+    const float cosOpeningAngleU(std::fabs(directionU.GetCosOpeningAngle(xAxis)));
+    const float cosOpeningAngleV(std::fabs(directionV.GetCosOpeningAngle(xAxis)));
+    const float cosOpeningAngleW(std::fabs(directionW.GetCosOpeningAngle(xAxis)));
+
+    const float xSeparation(5.f);
```

Comment:
Should this be configurable?

Reply:
Other than clarifying this, looks good to me too.

Reply:
sorted